tests/test_ptable.py

```diff
@@ -506,39 +555,47 @@ def test_ptable_heatmap_plotly_label_map(
 
 
 @pytest.mark.parametrize(
-    "data, symbol_pos, anno_kwds, hist_kwds",
```

Comment:
Need confirmation on this change.

Previously the `ptable_hists` has both `symbol_kwargs` and `anno_kwds`. In the new implementation, support for `annotate` is completely removed.

I'm not sure if we need the `annotate` in the first place, because it appears we just need a `symbol` to show for each element tile (users could tweak the text from `symbol_text`), we don't seem to need another annotation, which might be too small to show properly anyway. 

Can you please suggest on this?@janosh

Reply:
it works for me locally. maybe try updating your `matplotlib`? i restored the `mpl.colors.to_rgb` part in order to apply an alpha channel. i think background colors need to be applied as a light touch only, otherwise the plot becomes unreadable.
![image](https://github.com/janosh/pymatviz/assets/30958850/6eb2fbdb-adde-476c-be4d-d784e33ebda5)

**without alpha channel**:

![72b8caff-2ee6-4651-bd30-4797ea4d5a3c](https://github.com/janosh/pymatviz/assets/30958850/2fb3119d-bd57-4860-af11-13b920c991e7)

Reply:
Yes this `alpha` looks indeed a must. Don't know about why, would have another try tomorrow. Thanks!

Reply:
Update on this one, `plt.show()` seems to work fine, but `save_and_compress_svg` saved the figure without background color. I'm looking into it.

Reply:
oh, that's good to know. maybe we should change the default `save_and_compress_svg(transparent=False)`?